databases/week2/homework_4_again.sql

```diff
@@ -0,0 +1,46 @@
+set names utf8mb4;
```

Comment:
This is very good! :)

Just one tip about good practices, the SQL keywords should always be in uppercase, for instance:

```
CREATE TABLE users (
    id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
    username VARCHAR(255) NOT NULL
    created_date DATE NOT NULL,
    updated_date DATE NOT NULL
)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Reply:
ok I will do it next time